recipe/meta.yaml

```
@@ -35,7 +36,6 @@ test:
   commands:
     - pip check
     - macho_dump --help
```

```suggestion
    - macho_dump --help
    - macho_standalone --help
    - macho_find --help
```
Could you try if all commands work?

---

Absolutely! Thanks!

---

These commands are all deprecated! I will be reverting and adding the previous test commands!